src/views/components/LogConsole.mxml

```diff
@@ -0,0 +1,163 @@
+            VideoAd.getInstance().removeEventListener(HeyzapEvent.DID_HIDE, onVideoEvent);
+            VideoAd.getInstance().removeEventListener(HeyzapEvent.DID_FAIL, onVideoEvent);
+            VideoAd.getInstance().removeEventListener(HeyzapEvent.IS_AVAILABLE, onVideoEvent);
+            VideoAd.getInstance().removeEventListener(HeyzapEvent.FETCH_FAILED, onVideoEvent);
+            VideoAd.getInstance().removeEventListener(HeyzapEvent.ON_AUDIO_STARTED, onVideoEvent);
+            VideoAd.getInstance().removeEventListener(HeyzapEvent.ON_AUDIO_FINISHED, onVideoEvent);
+
+            // INCENTIVIZED CALLBACKS
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.DID_SHOW, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.DID_CLICK, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.DID_HIDE, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.DID_FAIL, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.ON_AUDIO_STARTED, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.ON_AUDIO_FINISHED, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.IS_AVAILABLE, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.FETCH_FAILED, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.IS_COMPLETED, onIncentivizedAdEvent);
+            IncentivizedAd.getInstance().removeEventListener(HeyzapEvent.IS_INCOMPLETE, onIncentivizedAdEvent);
+
+        }
+
+        private function onInterstitialEvent(event:HeyzapEvent):void {
+            showLine("[INTERSTITIAL] " + event.type);
+
+        }
+
+        private function onBannerEvent(event:HeyzapEvent):void {
+            showLine("[BANNER] " + event.type);
+
+            switch (event.type) {
+                case HeyzapEvent.ON_LOADED:
+                    var dim:Rectangle = BannerAd.getInstance().dimensions();
+                    if (dim != null) {
+                        showLine("Banner ad dimensions: x=" + dim.x + " y=" + dim.y + " width=" + dim.width + " height=" + dim.height);
+                    } else {
+                        showLine("Could not get banner ad dimensions.");
+                    }
+                    return;
+            }
+        }
+
+        private function onVideoEvent(event:HeyzapEvent):void {
+            showLine("[VIDEO] " + event.type);
+
+        }
+
+        private function onIncentivizedAdEvent(event:HeyzapEvent):void {
+            showLine("[INCENTIVIZED] " + event.type);
+
+            switch (event.type) {
+                case HeyzapEvent.IS_AVAILABLE:
+                    if (IncentivizedAd.getInstance().isAvailable()) {
```

Comment:
the `IS_AVAILABLE` and `FETCH_FAILED` events are opposites, and are called in response to `fetch`es; you should not have to check `IncentivizedAd.getInstance().isAvailable()` here, as it should always be true (although since you're not passing a tag in this check, it wouldn't necessarily always be true).